server/module/services/service/inference_service.py

```diff
@@ -414,48 +414,42 @@ async def run_tts_triton_inference(
         if format == "pcm":
             format = "s16le"
 
-        results = []
-
-        for input in request_body.input:
-            input_string = input.source.replace("।", ".").strip()
-
-            if input_string:
-                inputs, outputs = get_tts_io_for_triton(
-                    input_string, ip_gender, ip_language
-                )
+        inputs_ = [input.source.replace("।", ".").strip() for input in request_body.input]
```

Comment:
Just now noticed this. Since `replace("।", ".")` is something specific to coqui, can we please move this under the TTS repo's preprocess function?

Reply:
Cool